src/main/java/frc/robot/led/LED.java

```diff
@@ -0,0 +1,77 @@
+package frc.robot.led;
+
+import static frc.robot.led.LEDConstants.BLACK;
+import static frc.robot.led.LEDConstants.MAX_HUE;
+import static frc.robot.led.LEDConstants.RAINBOW_INDICATOR;
+import static frc.robot.led.LEDConstants.RAINBOW_SATURATION;
+import static frc.robot.led.LEDConstants.RAINBOW_VALUE;
+import static frc.robot.led.LEDConstants.STRIP_LENGTH;
+
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class LED extends SubsystemBase {
+
+    private Color currentColor;
+
+    private final LEDComponents components;
+
+    private LED(LEDComponents components) {
+        this.components = components;
+    }
+
+    public String getCurrentColor() {
+        if (currentColor == null)
+            return "no color";
+        return "red: " + currentColor.red +
+                " green: " + currentColor.green +
+                " blue" + currentColor.blue;
+    }
+
+    private void setOneLed(int ledIndex, Color color) {
+        components.getBuffer().setLED(ledIndex, color);
+    }
+
+    public void setStrip(Color color) {
+        for (int ledIndex = 0; ledIndex < STRIP_LENGTH; ++ledIndex)
+            setOneLed(ledIndex, color);
+
+        this.currentColor = color;
+        Update();
+    }
+
+    public void setStripOff() {
```

Comment:
`turnOff` is better name, you dont set the strip to off, you set the strip to black values.